.travis.yml

```diff
@@ -0,0 +1,469 @@
+  fi
+
+# Unit tests and dynamic linting
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- if [[ $TRAVIS_TAG ]]; then
+    conda build -q tools/conda.recipe &&
+    conda install --use-local ${CONDA_PKG_NAME_PY} &&
+    (cd; nosetests ${PROJECT_NAME} -v --detailed-errors) &&
+    conda uninstall ${CONDA_PKG_NAME_PY};
+  else
+    python setup.py build_ext -i &&
+    nosetests ${PROJECT_NAME}
+         -v --detailed-errors --with-coverage --cover-package=${PROJECT_NAME}
+         --cover-tests --cover-inclusive --cover-branches &&
+    coverage xml -i &&
+
+    if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
+       cardboardlinter --refspec $TRAVIS_BRANCH -f 'dynamic';
+    fi
+  fi
+
+# Make CPP and PY source package for github deployment
+- if [[ $TRAVIS_TAG ]]; then
+    python setup.py sdist;
+  fi
+
+# Some other stuff
+# ----------------
+
+# Compile documentation
+- if [[ -e doc ]]; then
+    (cd doc; make html);
+  fi
+
+after_success:
+# Upload the coverage analysis
+- codecov -f coverage.xml
+
+before_deploy:
+# Try to set some env vars to configure deployment.
+# Please keep the following lines. They will be used again as soon as it is supported on
+# travis. See https://github.com/travis-ci/dpl/issues/613
+#- export IS_PRERELEASE=$(python -c 'import os; tt=os.environ["TRAVIS_TAG"]; print("true" if ("a" in tt or "b" in tt) else "false")')
+#- echo ${IS_PRERELEASE}
+  - export ANACONDA_LABEL_ARG=$(sed -e 's/.*a.*/dev/' -e 's/.*b.*/test/' -e 's/.*.*/main/' <<< ${TRAVIS_TAG})
+  - echo ${ANACONDA_LABEL}
+
+# In deployment, the env var TRAVIS_TAG contains the name of the current version, if any.
+deploy:
+- provider: releases
+  skip_cleanup: true
+  api_key: ${GITHUB_TOKEN}
+  file: dist/${PROJECT_NAME}-${TRAVIS_TAG}.tar.gz
+  on:
+    repo: ${GITHUB_REPO_NAME}
+    tags: true
+    condition: "$MYCONDAPY == 2.7 && $TRAVIS_OS_NAME == linux && $TRAVIS_TAG == *[ab]*"
+  prerelease: true
+
+- provider: releases
+  skip_cleanup: true
+  api_key: ${GITHUB_TOKEN}
+  file: dist/${PROJECT_NAME}-${TRAVIS_TAG}.tar.gz
+  on:
+    repo: ${GITHUB_REPO_NAME}
+    tags: true
+    condition: "$MYCONDAPY == 2.7 && $TRAVIS_OS_NAME == linux && $TRAVIS_TAG != *[ab]*"
+  prerelease: false
+
+- provider: script
+  skip_cleanup: true
+  script: anaconda -t $ANACONDA_TOKEN upload --force -l ${ANACONDA_LABEL} ${HOME}/miniconda/conda-bld/*/${PROJECT_NAME}-*.tar.bz2
+  on:
+    repo: ${GITHUB_REPO_NAME}
+    tags: true
+
+- provider: pypi
+  skip_cleanup: true
+  user: ${PYPI_LOGIN}
+  password: ${PYPI_PASSWD}
+  on:
+    repo: ${GITHUB_REPO_NAME}
+    tags: true
+    condition: "$TRAVIS_TAG != *[ab]* && $MYCONDAPY == 2.7 && $TRAVIS_OS_NAME == linux"
+
+- provider: pages
+  skip_cleanup: true
+  github_token: ${GITHUB_TOKEN}
+  project_name: ${PROJECT_NAME}
+  local_dir: doc/_build/html
+  on:
+    repo: ${GITHUB_REPO_NAME}
+    condition: "-e doc && $TRAVIS_TAG != *[ab]* && $MYCONDAPY == 2.7 && $TRAVIS_OS_NAME == linux"
+    tags: true
+####
```

Comment:
Everything below this line is overriding the settings above this line. I think the lower half show be removed? Can you check this is caused by the cookie-cutter or if it was due to another edit?